persistence/src/main/scala/org/apache/pekko/persistence/state/javadsl/DurableStateUpdateStore.scala

```diff
@@ -26,12 +26,39 @@ import pekko.Done
 trait DurableStateUpdateStore[A] extends DurableStateStore[A] {
 
   /**
-   * @param seqNr sequence number for optimistic locking. starts at 1.
+   * Upsert the object with the given `persistenceId` and `revision`.
+   *
+   * @param persistenceId the persistenceId of the object to upsert
+   * @param revision the revision of the object to upsert
+   * @param value the value to upsert
+   * @param tag a tag to associate with the object
+   * @return a CompletionStage that completes when the object has been upserted
    */
   def upsertObject(persistenceId: String, revision: Long, value: A, tag: String): CompletionStage[Done]
```

Comment:
The generics type parameter name `A` is unusual.  https://docs.oracle.com/javase/tutorial/java/generics/types.html

But we could fixe it in another PR.